app/models/facility.rb

```diff
@@ -140,9 +140,7 @@ def dashboard_analytics(period: :month, prev_periods: 3, include_current_period:
      longitude: "longitude (optional)",
      facility_size: "size (optional)",
      enable_diabetes_management: "enable_diabetes_management (true/false)",
-     enable_teleconsultation: "enable_teleconsultation (true/false)",
-     teleconsultation_phone_number: "teleconsultation_phone_number",
```

Comment:
Is it ok to revoke support for this? Do we need to let anybody know?

Should we continue to accept phone numbers and gracefully associate the right users if they exist?

Reply:
These fields in the CSV are for the old old teleconsultation numbers, when we had one number per facility. We could let CVHOs submit one user's numbers in the template 🤷 , although it might be easier for them to upload the CSV and then add them all from the UI.

We will continue sending these fields in the app, so the API contract doesn't change.

Reply:
Hmm yeah tricky. I'd run this one by Praveen or Mahima to learn how uploads + telemed is being used, and at least let folks know that this is coming down the pike.